lua/autorun/server/bounties.lua

```diff
@@ -0,0 +1,87 @@
+util.AddNetworkString("Bounties")
+util.AddNetworkString("IncreaseBounty")
+
+hook.Add("PlayerDeath","PlayerDeath",function(ply,infl,atk)
+	local bounty = tonumber(ply:GetPData("bounty",0))
+	if bounty > 0 and ply != atk then
+		atk:PS_GivePoints(bounty)
+		BotSayGlobal("[edgy]"..atk:Nick().." [fbc]has claimed [gold]"..ply:Nick().."'s [fbc]bounty of [rainbow]"..bounty.." [fbc]points!")
+		ply:SetPData("bounty",0)
+	end
+end)
```

Comment:
I apologize if I'm missing something here, but are you checking to make sure that the attacker is a player? Just in case the player is killed by an entity, (like keemstar) you don't want to attempt to give the entity points

Reply:
Referring to the PlayerDeath hook

Reply:
you're right, I didn't read the hook's info close enough